lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -1492,7 +1492,7 @@ static bool sinkRescaleQuantizedNode(Function *F) {
       }
 
       auto *newReshape = F->createReshape(reshape->getName(),
-                                          rescale->getInput(), reshape->dims());
+                                          rescale->getInput(), reshape->getResult().dims());
```

Comment:
nit: make sure you run clang-format before merging

Reply:
damn :( forgot! will run real quick.

Reply:
we need to have a clang format checker before merge :) (seperate story).

https://github.com/pytorch/glow/pull/1253